<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit_calc</class>
 <widget class="QDialog" name="Credit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    font-family: 'Times New Roman', Times, serif;
    font-size: 20px;
    background-color:#484545
}


QLineEdit {
margin: 10px;
border-radius: 8px;
}

QLabel {
background-color: white;
border-radius: 8px;
margin: 10px;
border: 2px solid orange
}

QLabel#label_output {
margin: 15px;
margin-left: 40px;
margin-right: 40px
}

QPushButton {
margin-top: 12px;
 background-color: orange;
border-radius: 10px
}


QPushButton:pressed {
    background-color: white;
}

QComboBox {
margin:8px;
border-radius: 8px;
padding-left: 10px	
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layout_credit">
     <item>
      <widget class="QFrame" name="frame_layout">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QComboBox" name="combo_payment_type">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>120</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <item>
         <property name="text">
          <string>Аннуитетные</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Дифференцированные</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="line_edit_term">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>40</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_edit_interest_rate">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>80</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_edit_total_credit">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>0</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_total_credit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Сумма кредита (в рублях):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_term_credit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Срок кредита:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_interest_rate">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Процентная ставка:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_payment_type">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Тип платежа:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_term_credit">
        <property name="geometry">
         <rect>
          <x>362</x>
          <y>40</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <item>
         <property name="text">
          <string>В месяцах</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>В годах</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btn_calculate">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Рассчитать</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>511</width>
          <height>331</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>509</width>
           <height>329</height>
          </rect>
         </property>
         <widget class="QTableWidget" name="table_credit">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>471</width>
            <height>331</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>501</width>
            <height>331</height>
           </size>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
